src/media/js/views/homepage.js

```diff
@@ -179,67 +167,44 @@ define('views/homepage',
             return;
         }
 
-        this.classList.add('pressed');
-    });
-
-    z.page.on('keyup mouseup touchend', '.focusable', function(e) {
-        if (e.type === 'keyup' && e.keyCode !== KeyEvent.DOM_VK_RETURN) {
-            return;
-        }
-
-        this.classList.remove('pressed');
-        this.classList.add('released');
-
-        if (e.type === 'keyup') {
-            if (!caps.webApps) {
-                return;
-            }
-
-            // Preview current focused app.
-            var focusedApp = appsModel.lookup(this.dataset.id);
-
-            if (focusedApp.doc_type === 'webapp') {
-                var focusedManifestURL = focusedApp.manifest_url;
+        // Preview current focused app.
+        var focusedApp = appsModel.lookup(this.dataset.id);
 
-                apps.getInstalled().done(function(installedApps) {
-                    // Check if app is installed.
-                    var isInstalled = installedApps.some(function(installedManifestURL) {
-                        return focusedManifestURL === installedManifestURL;
-                    });
+        if (focusedApp.doc_type === 'webapp') {
+            var focusedManifestURL = focusedApp.manifest_url;
 
-                    if (isInstalled) {
-                        apps.launch(focusedManifestURL);
-                    } else {
-                        apps.install(focusedApp);
-                    }
+            apps.getInstalled().done(function(installedApps) {
+                // Check if app is installed.
+                var isInstalled = installedApps.some(function(installedManifestURL) {
+                    return focusedManifestURL === installedManifestURL;
                 });
-            } else {
-                window.open(focusedApp.url, '_blank',
-                    'remote=true,preview=true' +
-                    ',name=' + encodeURIComponent(focusedApp.name) +
-                    ',iconUrl=' + encodeURIComponent(findLargestIcon(focusedApp.icons)));
-            }
-        }
-    });
 
-    z.page.on('transitionend', '.focusable', function() {
-        if (this.classList.contains('released')) {
-            this.classList.remove('released');
+                if (isInstalled) {
+                    apps.launch(focusedManifestURL);
+                } else {
+                    apps.install(focusedApp);
+                }
+            });
+        } else {
+            window.open(focusedApp.url, '_blank',
+                'remote=true,preview=true' +
+                ',name=' + encodeURIComponent(focusedApp.name) +
+                ',iconUrl=' + encodeURIComponent(findLargestIcon(focusedApp.icons)));
         }
     });
 
-    z.page.on('blur', '.focusable', function() {
-        this.classList.remove('focused');
-        this.classList.remove('pressed');
-        this.classList.remove('released');
-    });
-
     return function(builder) {
+        if (!localStorage.getItem('marketplace.tutorial.fteskip')) {
```

Comment:
This will likely get reset with firmware updates but it's probably the desired behavior.